booru/admin.py

```diff
@@ -12,6 +12,9 @@
 from .models import Rating
 admin.site.register(Rating)
 
+from .models import Comment
+admin.site.register(Comment)
+
```

Comment:
Move this to the `comment` part of the file.  I.e. comment `# Comments` above it or something